python/lsst/ts/xml/data/sal_interfaces/ATBuilding/ATBuilding_Commands.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsl"?>
+<SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
+  <!-- The main commands -->
+  <SALCommand>
+    <Subsystem>ATBuilding</Subsystem>
+    <EFDB_Topic>ATBuilding_command_openVentGate</EFDB_Topic>
+    <Description>Open the specified vent gate.</Description>
+    <item>
+      <EFDB_Name>gate</EFDB_Name>
+      <Description>Which gate to open (0-3, counter-clockwise from door).</Description>
+      <IDL_Type>int</IDL_Type>
+      <Units>unitless</Units>
+      <Count>4</Count>
+    </item>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>ATBuilding</Subsystem>
+    <EFDB_Topic>ATBuilding_command_closeVentGate</EFDB_Topic>
+    <Description>Close the specified vent gate.</Description>
+    <item>
+      <EFDB_Name>gate</EFDB_Name>
+      <Description>Which gate to close (0-3, counter-clockwise from door).</Description>
+      <IDL_Type>int</IDL_Type>
+      <Units>unitless</Units>
+      <Count>4</Count>
+    </item>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>ATBuilding</Subsystem>
+    <EFDB_Topic>ATBuilding_command_extractionFanStart</EFDB_Topic>
```

Comment:
Consider making the "action" first like you did for the other command, e.g.; `ATBuilding_command_startExtractionFan`

Reply:
will do